torchgeo/datasets/idtrees.py

```diff
@@ -182,13 +181,6 @@ def __init__(
         self.num_classes = len(self.classes)
         self._verify()
 
-        try:
```

Comment:
It wasn't easy to test missing pyvista when laspy is also missing without this change.

Reply:
Same here -- I think the dataset should fail to instantiate if the deps aren't installed

Reply:
I think I have an idea that will allow us to fail-fast and still get full coverage, will open another prereq PR with just that change so that this PR can be a test-only PR.